src/args.rs

```diff
@@ -78,17 +78,17 @@ pub(crate) fn clap_app(tmp_dir: &str) -> clap::App {
         .arg(
             Arg::with_name(FTPS_CERTS_FILE)
                 .long("ftps-certs-file")
-                .value_name("PKCS12_ARCHIVE")
-                .help("Sets the path the the certificates used for TLS security/FTPS")
+                .value_name("PEM_FILE")
```

Comment:
This is not a `PEM_FILE` but a `CERTS-FILE`

Reply:
Its PEM format. We could also name it PEM_FORMATTED_CERTS_FILE

Reply:
I opted for this (FILE and PEM mentioned in the help):

```
        .arg(
            Arg::with_name(FTPS_CERTS_FILE)
                .long("ftps-certs-file")
                .value_name("FILE")
                .help("Sets the path the the certificates (PEM format) used for TLS security")
                .env("UNFTP_FTPS_CERTS_FILE")
                .takes_value(true),
        )
```